packages/flutter_tools/test/commands.shard/hermetic/run_test.dart

```diff
@@ -1335,13 +1335,10 @@ class FakeIOSDevice extends Fake implements IOSDevice {
 
 class TestRunCommandForUsageValues extends RunCommand {
   TestRunCommandForUsageValues({
-   this.devices,
-  });
-
-  @override
-  // devices is not set within usageValues, so we override the field
-  // ignore: overridden_fields
-  List<Device>? devices;
+   List<Device>? devices,
```

Comment:
can this be in an initializer list, or must it be a constructor body?

Also indentation nit (even though it already exists before this PR)
```suggestion
    List<Device>? devices,
```

Reply:
It has to be in the body because the super class does not expose a constructor parameter. This is a weird case because of it being a testing only thing, usually this would be assigned asynchronously later, but that doesn't happen in the testing context.

Reply:
makes sense (sort of :P )